### PR TITLE
code fixes at infinitum

### PR DESCRIPTION
**Pull request by Evankhell561**
before:
image

after:
image

### CODE REVIEW

File: code/modules/admin/admin.dm [hunk 1]
```diff
@@ -1175,9 +1175,8 @@ datum/admins/var/obj/item/weapon/paper/admin/faxreply // var to hold fax replies
 		P.stamps += "<hr><i>This paper has been stamped by the [P.origin] Quantum Relay.</i>"
 
 		var/image/stampoverlay = image('icons/obj/bureaucracy.dmi')
-		var/{x; y;}
```
Comment (NanakoAC) [inline code/modules/admin/admin.dm:1178]: this is some wierd syntax, are you sure its not valid?

File: code/modules/mob/living/carbon/human/species/necromorph/ubermorph.dm [hunk 2]
```diff
@@ -277,7 +277,7 @@ Best used near the end, when all seems quiet, to help the necromorphs hunt down
 			//Find target organ should never fail, we won't bother checking
 			targetstring = "[H]'s [found_organ]"
 
-			playsound(charge., 'sound/weapons/slice.ogg', VOLUME_MAX, 1)
```
Comment (NanakoAC) [inline code/modules/mob/living/carbon/human/species/necromorph/ubermorph.dm:280]: good catch, wrong fix. This sound should be played from charge.victim

charge is a non atomic datum here

**Review by NanakoAC — CHANGES_REQUESTED** [review]
just one fix with the ubermorph

File: code/modules/admin/admin.dm (hunk 1 above)
Comment (Evankhell561) [reply code/modules/admin/admin.dm:1178]: Byond may allow compiling in this format but I try to make everything conform to the standard to which we write code.

**Pull request merged**